jest.json

```
@@ -18,14 +18,14 @@
         "jest/no-restricted-matchers": [
           "error",
           {
-            "toMatchSnapshot": null,
-            "toMatchInlineSnapshot": null,
-            "toThrowErrorMatchingSnapshot": null,
-            "toThrowErrorMatchingInlineSnapshot": null,
-
-            "toBeTruthy": null,
-            "toBeFalsy": null,
-            "toBeDefined": null
+            "toMatchSnapshot": "Use more targeted assertions instead of a snapshot. See https://docs.gitlab.com/ee/development/testing_guide/frontend_testing.html#snapshots",
+            "toMatchInlineSnapshot": "Use more targeted assertions instead of a snapshot. See https://docs.gitlab.com/ee/development/testing_guide/frontend_testing.html#snapshots",
+            "toThrowErrorMatchingSnapshot": "Use more targeted assertions instead of a snapshot. See https://docs.gitlab.com/ee/development/testing_guide/frontend_testing.html#snapshots",
+            "toThrowErrorMatchingInlineSnapshot": "Use more targeted assertions instead of a snapshot. See https://docs.gitlab.com/ee/development/testing_guide/frontend_testing.html#snapshots",
+
+            "toBeTruthy": "Use more explicit matchers like .toBe(true), .toBeGreaterThan(0), or .not.toBe(undefined). See https://docs.gitlab.com/ee/development/testing_guide/frontend_testing.html#avoid-using-tobetruthy-or-tobefalsy",
```

```suggestion
            "toBeTruthy": "Use more explicit matchers like `.toBe(true)` or `.toBeGreaterThan(0)`. See https://docs.gitlab.com/ee/development/testing_guide/frontend_testing.html#avoid-using-tobetruthy-or-tobefalsy",
```

`.not.toBe(undefined)` is even worse than `toBeTruthy()`, it's loose **and** confusing.